.travis.yml

```diff
@@ -1,3 +1,5 @@
 language: node_js
 node_js:
-  - '9'
+  - '10'
+  - '12'
```

Comment:
Maybe worth explicitly adding 13 as well?

Reply:
Sure

Reply:
`node` or `latest` may be better than `13` (fail early).

Reply:
I added both